diff --git a/node_modules/react-native-material-dropdown/src/components/dropdown/index.js b/node_modules/react-native-material-dropdown/src/components/dropdown/index.js
index b1e65a1..66a13a8 100644
--- a/node_modules/react-native-material-dropdown/src/components/dropdown/index.js
+++ b/node_modules/react-native-material-dropdown/src/components/dropdown/index.js
@@ -229,7 +229,7 @@ export default class Dropdown extends PureComponent {
       event.nativeEvent.locationX -= this.rippleInsets().left;
 
       /* Start ripple directly from event */
-      this.ripple.startRipple(event);
+      // this.ripple.startRipple(event);
     }
 
     if (!itemCount) {
@@ -481,6 +481,7 @@ export default class Dropdown extends PureComponent {
       renderBase,
       labelExtractor,
       dropdownOffset,
+      styleInput,
       renderAccessory = this.renderAccessory,
     } = this.props;
 
@@ -507,9 +508,8 @@ export default class Dropdown extends PureComponent {
       <TextField
         label=''
         labelHeight={dropdownOffset.top - Platform.select({ ios: 1, android: 2 })}
-
+        styleInput
         {...props}
-
         value={title}
         editable={false}
         onChangeText={undefined}
@@ -550,13 +550,13 @@ export default class Dropdown extends PureComponent {
   }
 
   renderAccessory() {
-    let { baseColor: backgroundColor } = this.props;
+    let { baseColor: backgroundColor, CustomArrow , arrowStyles} = this.props;
     let triangleStyle = { backgroundColor };
 
     return (
       <View style={styles.accessory}>
         <View style={styles.triangleContainer}>
-          <View style={[styles.triangle, triangleStyle]} />
+          <CustomArrow isEnabled={this.state.modal}/>
         </View>
       </View>
     );
@@ -727,12 +727,10 @@ export default class Dropdown extends PureComponent {
     return (
       <View onLayout={this.onLayout} ref={this.updateContainerRef} style={containerStyle}>
         <TouchableWithoutFeedback {...touchableProps}>
-          <View pointerEvents='box-only'>
+          <View>
             {this.renderBase(props)}
-            {this.renderRipple()}
           </View>
         </TouchableWithoutFeedback>
-
         <Modal
           visible={modal}
           transparent={true}
diff --git a/node_modules/react-native-material-dropdown/src/components/dropdown/styles.js b/node_modules/react-native-material-dropdown/src/components/dropdown/styles.js
index 39031e0..9c96177 100644
--- a/node_modules/react-native-material-dropdown/src/components/dropdown/styles.js
+++ b/node_modules/react-native-material-dropdown/src/components/dropdown/styles.js
@@ -19,8 +19,8 @@ export default StyleSheet.create({
   },
 
   triangleContainer: {
-    width: 12,
-    height: 6,
+    width: 30,
+    height: 30,
     overflow: 'hidden',
     alignItems: 'center',
 
